src/main/java/com/blackducksoftware/integration/jira/issue/handler/JiraIssueHandler.java

```diff
@@ -286,66 +287,79 @@ private void addComment(final BlackDuckIssueModel blackDuckIssueModel, final Str
-            } catch (final JiraIssueException e) {
+            } catch (JiraIssueException e) {
                 handleJiraIssueException(e, blackDuckIssueModel);
             }
         }
     }
 
-    private void updateIssueTrackerProperties(final BlackDuckIssueModel blackDuckIssueModel, final Issue issue) {
-        final String blackDuckIssueUrl = issueTrackerHandler.createBlackDuckIssue(blackDuckIssueModel.getComponentIssueUrl(), issue);
+    private void updateIssueTrackerProperties(BlackDuckIssueModel blackDuckIssueModel, Issue issue) {
+        String blackDuckIssueUrl = issueTrackerHandler.createBlackDuckIssue(blackDuckIssueModel.getComponentIssueUrl(), issue);
         if (StringUtils.isNotBlank(blackDuckIssueUrl)) {
-            final IssueTrackerProperties issueTrackerProperties = new IssueTrackerProperties(blackDuckIssueUrl, blackDuckIssueModel.getJiraIssueId());
+            IssueTrackerProperties issueTrackerProperties = new IssueTrackerProperties(blackDuckIssueUrl, blackDuckIssueModel.getJiraIssueId());
             try {
-                final String key = IssueTrackerHandler.createEntityPropertyKey(blackDuckIssueModel.getJiraIssueId());
+                String key = IssueTrackerHandler.createEntityPropertyKey(blackDuckIssueModel.getJiraIssueId());
                 issueServiceWrapper.getIssuePropertyWrapper().addProjectProperty(blackDuckIssueModel.getJiraIssueFieldTemplate().getProjectId(), jiraUserContext.getJiraAdminUser(), key, issueTrackerProperties);
-            } catch (final JiraIssueException e) {
+            } catch (JiraIssueException e) {
                 handleJiraIssueException(e, blackDuckIssueModel);
             }
         }
     }
 
-    private void updateDefaultIssueProperties(final BlackDuckIssueModel blackDuckIssueModel) {
-        final BlackDuckIssueFieldTemplate blackDuckIssueFieldTemplate = blackDuckIssueModel.getBlackDuckIssueFieldTemplate();
-        final IssueCategory category = blackDuckIssueFieldTemplate.getIssueCategory();
+    private void updateDefaultIssueProperties(BlackDuckIssueModel blackDuckIssueModel) {
+        BlackDuckIssueFieldTemplate blackDuckIssueFieldTemplate = blackDuckIssueModel.getBlackDuckIssueFieldTemplate();
+        IssueCategory category = blackDuckIssueFieldTemplate.getIssueCategory();
         if (!IssueCategory.SPECIAL.equals(category)) {
-            final IssueProperties properties = new IssueProperties(category, blackDuckIssueModel.getBomComponentUri(), blackDuckIssueFieldTemplate.getPolicyRuleName(), blackDuckIssueModel.getJiraIssueId());
+            IssueProperties properties = new IssueProperties(category, blackDuckIssueModel.getBomComponentUri(), blackDuckIssueFieldTemplate.getPolicyRuleName(), blackDuckIssueModel.getJiraIssueId());
+
+            String alertCategory;
```

Comment:
Since we don't use final anymore, we could set this equal to Policy then just do a basic if statement to change it if necessary.